### DISCUSSION TITLE
⭐ 200 stars ⭐

### DISCUSSION THREAD

**Discussion by LouisGariepy** [cornucopia-rs/cornucopia#188] [Announcements]
We just reached 200 stargazers on our main repo! It might seem small to some of you, but its the first project I'm involved in that has gained any form of traction.

A big thanks to all our contributors, old and new, big and small, Cornucopia wouldn't be possible without you.

I'd also like to thank our users. Not only is it motivating to see Cornucopia used in action, but you've also helped improve the project with your suggestions.

I'm confident that Cornucopia will only get better from here, with lots of big features being already on the way.

Cheers! :rocket:  :star:

**Comment by jacobsvante** [comment]
To the stars and beyond! I think this repo deserves many more still 😊

**Comment by jacobsvante** [comment]
Thank you once again for creating this!
